modules/01-filter.js

```diff
@@ -9,9 +9,10 @@ const Augur = require("augurbot"),
 const bannedWords = new RegExp(banned.words.join("|"), "i"),
   bannedLinks = new RegExp(`\\b(${banned.links.join("|").replaceAll(".", "\\.")})`, "i"),
   hasLink = /http(s)?:\/\/(\w+(-\w+)*\.)+\w+/,
-  pf = new profanityFilter(),
   scamLinks = new RegExp(`\\b(${banned.scam.join("|").replaceAll(".", "\\.")})`, "i");
 
+let pf = new profanityFilter();
```

Comment:
Let's move this line into the function so we don't have a global object sitting around taking up memory here.